src/Tizen.NUI.Components/Controls/ItemsView/ItemsView.cs

```diff
@@ -0,0 +1,454 @@
+        protected virtual ViewItem PopRecycleCache(DataTemplate Template)
+        {
+           for (int i = 0; i < RecycleCache.Count; i++)
+           {
+               ViewItem item = RecycleCache[i];
+               if (item.Template == Template)
+               {
+                   RecycleCache.Remove(item);
+                   item.Show();
+                   return item;
+               }
+           }
+           return null;
+        }
+
+        private void OnItemRelayout(object sender, EventArgs e)
+        {
+            ItemsLayouter.NotifyItemSizeChanged((sender as ViewItem));
+            //ItemsLayouter.RequestLayout(ScrollingDirection == Direction.Horizontal ? ContentContainer.CurrentPosition.X : ContentContainer.CurrentPosition.Y);
+        }
+
+        private void DecorateItem(ViewItem item, int Index)
+        {
+            item.Index = Index;
+            item.ParentItemsView = this;
+            /* -- Template and BindinContext Need to be implemented in child class.
+            item.Template = (ItemsTemplate is DataTemplateSelector ?
+                             (ItemsTemplate as DataTemplateSelector).SelectDataTemplate(InternalItemSource.GetItem(Index), this) :
+                              ItemsTemplate);
+               item.BindingContext = InternalItemSource.GetItem(Index);
+            item.BindingContext = InternalItemSource.GetItem(Index);
+            */
+            item.Relayout += OnItemRelayout;
+            ContentContainer.Add(item);
+            /* -- This seems wrong as calling multiple times while scrolling
+            PropertyNotification noti = item.AddPropertyNotification("size", PropertyCondition.Step(0.1f));
+            noti.Notified += OnItemSizeChanged;
+            notifications.Add(noti);
+            */
+        }
+
+        /// <summary>
+        /// Realize indexed item.
+        /// </summary>
+        /// <param name="Index"> Index position of realizing item </param>
+        /// <since_tizen> 8 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        internal virtual ViewItem RealizeItem(int Index)
+        {
+            // Check DataTemplate is Same!
+            if (ItemsTemplate is DataTemplateSelector)
+            {
+               Console.WriteLine("LSH :: DataTemplate is Selector??");
+            }
+            else
+            {
+               // pop item
+               ViewItem item = PopRecycleCache(ItemsTemplate);
+               if (item != null)
+               {
+                    DecorateItem(item, Index);
+                    return item;
+               }
+            }
+
+            object content = ItemsTemplate.CreateContent();
+            if (content == null)
+            {
+                Console.WriteLine("LSH :: Template CreateContent Failed!!!");
+            }
+            if (content as ViewItem)
```

Comment:
형식 테스트를 위해 `as`대신 `is` 사용을 권장합니다.
`if (content as ViewItem)`은 `if (content is ViewItem ? (ViewItem)content : (ViewItem)null)`의 동작과 동일합니다.

```suggestion
            if (content is ViewItem)
```